subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -123,6 +125,10 @@ abstract class InstrumentationTestsTask @Inject constructor(
             reportConfig
         )
 
+        val proguardMappings = project.tasks.withType<ProguardConfigurableTask>()
```

Comment:
Should we filter only tasks from a build variant in `android.testBuildType`?

Reply:
Fixed